src/reducers/todo.ts

```diff
@@ -38,6 +38,12 @@ const todoReducer = (state = initialState, action: any) => {
         ),
       };
 
+    case ADD_TODO:
+      return {
+        ...state,
+        todos: [...state.todos, action.todo],
```

Comment:
We can use immer instead it this. (No need to make changes in this PR. )

Reply:
yup 👍